### PR TITLE
Updated documentation --nomigrations to --no-migrations

### PR DESCRIPTION
**Pull request by etimoz**
Updated argument `--nomigrations` to `--no-migrations` in documentation according to `pytest --help`.

### CODE REVIEW

File: docs/database.rst [hunk 1]
```diff
@@ -121,13 +121,13 @@ A good way to use ``--reuse-db`` and ``--create-db`` can be:
 * When you alter your database schema, run ``pytest --create-db``, to force
   re-creation of the test database.
 
-``--nomigrations`` - Disable Django migrations
-----------------------------------------------
+``--no-migrations`` - Disable Django migrations
+-----------------------------------------------
 
-Using ``--nomigrations`` will disable Django migrations and create the database
+Using ``--no-migrations`` will disable Django migrations and create the database
```
Comment (bluetech) [inline docs/database.rst:127]:
```suggestion
Using ``--no-migrations`` (alias: ``--nomigrations``) will disable Django migrations and create the database
```

**Review by bluetech — APPROVED** [review]
It seems that both are accepted, so I added a little suggestion to mention this, in case someone does a search or gets confused, let me know what you think. But otherwise LGTM, thanks!

**Comment by etimoz** [comment]
yeah good idea to add the alias!

**Review by bluetech — APPROVED** [review]
Thanks!

**Pull request merged**